pkg/scheduler/cache/interface.go

```diff
@@ -80,7 +80,7 @@ type VolumeBinder interface {
 
 //Binder interface for binding task and hostname
 type Binder interface {
-	Bind(kubeClient *kubernetes.Clientset, tasks []*api.TaskInfo) (error, []*api.TaskInfo)
```

Comment:
Need to consider compatibility. The change will impact the bind implementation in user side.

Reply:
If not fix like this, the code verify will not be passed.